### PR TITLE
gcs: Include Gateway EUI on configuration

### LINKED ISSUE #1809: Include gateway_ID on GatewayConfigurationServer response
#### Summary

The configuration provided by GCS for the semtechudp packet forwarder does not include any gateway EUI information. Some versions of the packet forwarder (e.g. the one used in NASys gateways). As a result, even though the gateway may be able to connect to The Things Stack, the log messages show:

```
stack_1     | INFO Connected                                gateway_eui=0000000000000000 gateway_uid=my_gtw namespace=gatewayserver/io/udp protocol=udp
```

This, in turn, may cause errors later, because of possible EUI mismatch on different components.

#### Steps to Reproduce

1. Create a new gateway
2. curl https://thethings.example.com/api/v3/gcs/gateways/gtw1/semtechudp/global_conf.json

#### What do you see now?

```json

	"gateway_conf": {
		"server_address": "localhost",
		"serv_port_up": 1700,
		"serv_port_down": 1700,
		"servers": [
			{
				"server_address": "localhost",
				"serv_port_up": 1700,
				"serv_port_down": 1700,
				"serv_enabled": true
			}
		]
	}
```

#### What do you want to see instead?

```
	"gateway_conf": {
		"gateway_ID": "1212121212121121",
		"server_address": "localhost",
		"serv_port_up": 1700,
		"serv_port_down": 1700,
		"servers": [
			{
				"gateway_ID": "1212121212121121",
				"server_address": "localhost",
				"serv_port_up": 1700,
				"serv_port_down": 1700,
				"serv_enabled": true
			}
		]
	}

```

#### Environment

all

#### How do you propose to implement this?

Add a new field on the `GatewayConf` struct

#### Can you do this yourself and submit a Pull Request?

Yes, need review

### PR DESCRIPTION
**Pull request by neoaggelos**
#### Summary

Closes #1809 

#### Changes

- Include `gateway_ID` line for UDP packet forwarder configuration response

#### Notes for Reviewers

- Would this change need a minor version bump? It does not break compatibility, as this is the standard field name.

#### Checklist

- [X] Scope: The referenced issue is addressed, there are no unrelated changes.
- [X] Compatibility: The changes are backwards compatible with existing API, database and configuration, according to the stability commitments in `README.md`.
- [ ] Testing: The changes are covered with unit tests. The changes are tested manually as well.
- [ ] Documentation: Relevant documentation is added or updated.
- [X] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [X] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

**Comment by johanstokking** [comment]
What is the status here? Is this draft? Who is reviewing?

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Forwarding of backend warnings to the Console.
 - Auth Info service to the JavaScript SDK.
 - Subscribable events to the JavaScript SDK.
+- Include `gateway_ID` field in UDP configuration response from Gateway Configuration Server.
```
Comment (rvolosatovs) [inline CHANGELOG.md]:
```suggestion
- Include `gateway_ID` field in Semtech UDP configuration response from Gateway Configuration Server.
```

**Comment by neoaggelos** [comment]
> Can you link the "standard" where this field is defined?

Possibly misused the word "standard" here. From my experience, Kerlink and Nasys gateways use `gateway_ID`. It is `gateway_ID` on the [git repo of the packet forwarder](https://github.com/Lora-net/packet_forwarder/blob/master/lora_pkt_fwd/global_conf.json) as well.

**Comment by rvolosatovs** [comment]
I'm not experienced with Semtech gateways, but LGTM.
Please wait for @KrishnaIyer approval for merging though.

**Comment by htdvisser** [comment]
The reasoning behind omitting the Gateway EUI from the response was that that the EUI belongs in `local_conf.json` instead of `global_conf.json` (which we are serving here). In addition to this, the network shouldn't be telling a gateway what it's EUI is. The gateway should already know this.

**Pull request merged**